card/mod/05_standard/lib/file_uploader.rb

```diff
@@ -146,7 +146,7 @@ def retrieve_dir
   end
 
   def mod_file?
-    @mod ||= model.mod_file?
+    model.mod_file?
```

Comment:
This was the reason that if you update a mod file the file was stored in the wrong place (in the mod instead of the files directory). It doesn't seem very expensive to me. It checks regular expressions on the card's content